src/lib/selection-impl.ts

```diff
@@ -231,20 +295,18 @@ export class SelectionImpl<T> implements Selection<T> {
         } else {
           const {exitCallback, bindCallback} = this;
 
-          if (exitCallback || bindCallback) {
```

Comment:
This was a bug before. If the API user had not specified either an `exitCallback` nor a `bindCallback`, then exiting sprites would not have actually been dropped. By removing this `if` check, the `sprite.exit()` call will always occur, which has the side effect of releasing the Sprite memory for future use (eventually).